### PR TITLE
React-Redux Capstone PR

### PR DESCRIPTION
**Pull request by this-ezzy**
In this milestone, I was able to complete the following requirements

- Select the API I am going to use.
- Build files structure for my React app.
- Prepare routes and navigation in my app.
- Display a list of phone brands from the api
- Display a list of phones depending on the brands selected
- Display specs of selected phone
- Setup test with jest and react testing library
- Ensured no linter errors

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          2023
+          <span>Phone Specs</span>
+        </h3>
+        <h4 className="flex flex-col items-center">
+          {`${brands.length}
+          Brands`}
+        </h4>
+      </article>
+      <h3 className="text-white px-3 py-1 border bg-rose-500 border-[#e0407d] text-xl shadow-xl">
+        All Brands Stats
```
Comment (thecodechaser) [inline src/components/Home.jsx]: - [ ] You did really a great job on this project but you missed a major requirement. Please add a feature on the home page to filter Brands. (You should include a search feature on the page).
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Ezzywealth👋, 

### You are doing a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight 🎉
- All tests are passed. ✔️
- Nice UI design. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] The License link in the README file is broken, Please fix it to make your README file more professional.
- [ ] In the README file, Please add steps to run the project and to run the tests, for example. `npm run dev` and `npm run test`. It will make your README file more professional.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+  - Visit this link "https://github.com/Ezzywealth/React-Capstone"
+  - Click the green button labelled "code"
+  - Select the "Open with GitHub Desktop" option
+  - After the GitHub Desktop add opens, click the "clone repo" button
+
+### Usage
+
+To run the project, execute the following command: **npm install** **npm run
+dev**
```
Comment (DuaneDave) [inline README.md]: what the previous reviewer was trying to hint you on about the command for running test is that your readme lacks a section called run tests and it is in this section you'd include the command for running tests. Kindly include this command to make the readme more detailed.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Ezzywealth 👋 , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by TadesseDev — APPROVED** [review]
### Hi @Ezzywealth 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**